docs/getting-started.md

```diff
@@ -16,24 +16,31 @@ The API will reject you if you don't provide a `User-Agent` header (more details
 
 If you want to access private repositories or perform actions on behalf of a user, you need to pass credentials to the client.
 
-There are two options supported by the API - basic and OAuth authentication.
+There are three options supported by the API - Basic, Personal Access Token, and OAuth authentication.
```

Comment:
A personal access token _is_ an OAuth token.

How about we just say

> There are two options supported by the API - basic and OAuth (ex. personal access token) authentication.

Then there's no need to add anything else.